templates/http-zig/content/build.zig

```diff
@@ -0,0 +1,34 @@
+const std = @import("std");
+
+pub fn build(b: *std.build.Builder) void {
+    // Standard target options allows the person running `zig build` to choose
+    // what target to build for. Here we do not override the defaults, which
+    // means any target is allowed, and the default is native. Other options
+    // for restricting supported target set are available.
+    const target = b.standardTargetOptions(.{});
+
+    // Standard release options allow the person running `zig build` to select
+    // between Debug, ReleaseSafe, ReleaseFast, and ReleaseSmall.
+    const mode = b.standardReleaseOptions();
+
+    const exe = b.addExecutable("content", "src/main.zig");
+    exe.setTarget(target);
+    exe.setBuildMode(mode);
+    exe.install();
+
+    const run_cmd = exe.run();
```

Comment:
Not sure running the app makes sense, or am I misunderstanding what this does?   (I mean... I am clearly misunderstanding because you have done this...)

Reply:
This is a boilerplate Zig file generated by `zig init-exe`. I'm not necessarily sure if I should modify or edit it.

Reply:
I just removed the `build.zig`, as it does not seem to be required by the toolchain in order to build Wasm files